app/script/conversation/ConversationRepository.coffee

```diff
@@ -99,6 +99,7 @@ class z.conversation.ConversationRepository
     amplify.subscribe z.event.WebApp.SELF.CLIENT_ADD, @on_self_client_add
     amplify.subscribe z.event.WebApp.USER.UNBLOCKED, @unblocked_user
     amplify.subscribe z.event.WebApp.CONVERSATION.MESSAGE.DELETE, @delete_message
```

Comment:
Should we rename this to `z.event.WebApp.CONVERSATION.MESSAGE.DELETE_SELF` for additional clarity down the road?